### DISCUSSION TITLE
Replacement name in description is incorrect

### DISCUSSION THREAD

**Discussion by uhthomas** [renovatebot/renovate#23552] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Renovate states:

> This is a special PR that replaces `k8s.gcr.io-sig-storage-livenessprobe` with the community suggested minimal stable replacement version.

https://github.com/uhthomas/automata/pull/184

It should be `k8s.gcr.io/sig-storage/livenessprobe`.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment]
ok, it's just a cosmetic issue. we probably use the wrong placeholder inside the message

**Reply by viceice** [reply, marked as answer]
- #26034
